app/src/main/java/com/zeapo/pwdstore/git/GitOperation.kt

```diff
@@ -162,8 +160,9 @@ abstract class GitOperation(gitDir: File, internal val callingActivity: AppCompa
         return this
     }
 
-    private fun withOpenKeychainAuthentication(username: String, identity: SshApiSessionFactory.ApiIdentity?): GitOperation {
-        SshSessionFactory.setInstance(SshApiSessionFactory(username, identity))
+    private fun withOpenKeychainAuthentication(username: String, activity: AppCompatActivity): GitOperation {
```

Comment:
Make this take a `FragmentActivity`? Its one step below the inheritance chain and lets us call Git related things from fragments without needing to cast `requireActivity()` to `AppCompatActivity` every time.

Reply:
Done in the merge, thanks!